faithcomesbyhearing/dbp
# FCBH-2882 Social provider link account on login

# Description
Link social account on login
Create and link social account on login
## Issue Link
[FCBH-2882](https://fullstacklabs.atlassian.net/browse/FCBH-2882)

## How Do I QA This
Try to login with a social provider on the following 3 conditions and confirm are working
- New user
- User registered without any social account
- User registered with the social account

## Checklist
